src/main/webapp/globalVariables.js

```diff
@@ -18,6 +21,10 @@ let currentPage = 1;
 let next = true;
```

Comment:
Why do we need to keep "next" as global variable?
Can we pass it as an argument to changePage()?

Reply:
sure